### PR TITLE
Reference documentation for all /admin/projects/... endpoints

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

**Comment by rahul1** [comment]
Fixes #2545

File: packages/docs/docs/api/project-admin/members.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    reference: 'User/:userId',
+  },
+  profile: {
+    reference: 'Practitioner/:profileId',
+    display: 'George Washington'
+  },
+}
+```
+
+## POST `/admin/projects/:projectId/:membershipId`
```
Comment (codyebberson) [inline packages/docs/docs/api/project-admin/members.md:57]: Should be `/admin/projects/:projectId/members/:membershipId`

File: packages/docs/docs/api/project-admin/members.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    reference: 'User/:userId',
+  },
+  profile: {
+    reference: 'Practitioner/:profileId',
+    display: 'George Washington'
+  }
+}
+```
+
+## DELETE `/admin/projects/:projectId/:membershipId`
```
Comment (codyebberson) [inline packages/docs/docs/api/project-admin/members.md:175]: Should be `/admin/projects/:projectId/members/:membershipId`

**Review by codyebberson — COMMENTED** [review]
Looks good

Although... I wonder if we should actually formally deprecate the following endpoints: 
- `/admin/projects/:projectId`
- `/admin/projects/:projectId/:membershipId`
- `/admin/projects/:projectId/secrets`
- `/admin/projects/:projectId/sites`

Before, project admins did not have access to the underlying `Project` and `ProjectMembership` resources, so you had to use those specialty endpoints.  Now all of it can be done with `/fhir/R4/Project` and `/fhir/R4/ProjectMembership`, which is much cleaner and easier to understand, works with search, works with batch, works with graphql, etc.

If we do that, you could delete the following files:

- `project-admin/members.md`
- `project-admin/project.md`
- `project-admin/secrets.md`
- `project-admin/sites.md`

The other endpoints are good and necessary, because they do heavy lifting of creating memberships, setting up permissions, etc.

Your call if we should document them now, and clean it all up later, or if we should start deprecating now.

Otherwise, 2 small incorrect URL paths.

**Comment by rahul1** [comment]
If we don't need them, lets' formally deprecate. I'll remove the docs

**Review by codyebberson — APPROVED** [review]

**Pull request merged**